flows/runs/context.go

```diff
@@ -1,38 +1,31 @@
 package runs
 
 import (
-	"encoding/json"
 	"fmt"
 
 	"github.com/nyaruka/goflow/flows"
 	"github.com/nyaruka/goflow/utils"
 )
 
-// NewContextForContact creates a new context for the passed in contact
-func NewContextForContact(contact *flows.Contact, run flows.FlowRun) flows.Context {
-	context := context{contact: contact, run: run}
-	return &context
+type runContext struct {
```

Comment:
It's simpler to not store off the contact at this point because the contact might change